src/editors/containers/VideoUploadEditor/hooks.test.jsx

```diff
@@ -0,0 +1,108 @@
+import * as hooks from './hooks';
```

Comment:
Nit: new files should be using `.tsx` instead of `.jsx`, but it's OK here since this is an old PR and increasing our test coverage so well.